### ISSUE TITLE
Unnecessary Classes

### ISSUE THREAD

**Issue by zceyakr** [zachlehman99/more-css-practice#7]
While there is nothing wrong with putting a class on everything and using it that way, instead of using classes you could just use the element itself. This would save you time instead of typing the class for each individual element.

**Comment by zachlehman99** [comment]
Yeah, I sort of have to cut out many of my classes. I just need to work on that. It seems like I complicated things with adding lots of class.
